hypha/apply/projects/templates/application_projects/project_approval_detail.html

```diff
@@ -6,6 +6,14 @@
 {% block extra_css %}
         <link rel="stylesheet" href="{% static 'css/apply/fancybox.css' %}">
         {{ reviewer_form.media.css }}
+        <style>
```

Comment:
Added custom style because of rich text field questions and answers.
Rich text field questions have h4 tag([here](https://github.com/HyphaApp/hypha/pull/3476/files#diff-4003a4b96bb979d52a2799aa6becd989b9f1395b508186f32fbc6ea2a73377b1R91)), and couldn't update its tag because it is used at multiple places.

Reply:
Also, I would try avoid using h2/h3 just for the size, instead use it organize an article into different sections. Better to use the class names to keep the size consistent.

Reply:
Removed the custom style, and used classes to maintain size.